src/request/components/request-detail-panel-logging.tsx

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { ILogMessage } from '../messages/ILogMessage';
+import { ILoggingViewModel } from '../view-models/ILoggingViewModel';
+
+import React = require('react');
+
+interface ILogMessagesProps {
+    messages: ILogMessage[];
+}
+
+/**
+ * React class to display console messages
+ */
+class LogMessages extends React.Component<ILogMessagesProps, {}> {
+    public render() {
+        return (
+            <table className='table table-bordered table-striped tab-content-item'>
+                <thead>
+                    <tr className='table-col-title-group'>
+                        <th width='5%'><span className='table-col-title'>#</span></th>
+                        <th width='10%'><span className='table-col-title'>Level</span></th>
+                        <th><span className='table-col-title'>Message</span></th>
+                        <th width='10%'><span className='table-col-title'>From Start</span></th>
+                        <th width='10%'><span className='table-col-title'>Duration</span></th>
+                    </tr>
+                </thead>
+                {this.props.messages.map(function(message, index) {
+                    const className = LogMessages.getRowClass(message);
+
+                    return (
+                        <tr className={className}>
+                            <td>{index + 1}</td>
+                            <td>{message.level}</td>
+                            <td>{message.message}</td>
+                            <td>-</td>
+                            <td>-</td>
+                        </tr>);
+                }) }
+                <tfoot>
+                    <tr className='table-body-padding table-col-title-group'><th colSpan='5'></th></tr>
+                </tfoot>
+            </table>
+        );
+    }
+
+    /**
+     * Return the CSS class name to use for the given message
+     */
+    private static getRowClass(message: ILogMessage) {
+        'use strict';
+
+        let rowClass = 'tab-logs-data-default';
+        switch (message.level) {
+            case 'Verbose':
+            case 'Info':
+                rowClass = 'tab-logs-data-default';
+                break;
+            case 'Critical':
+            case 'Error':
+                rowClass = 'tab-logs-data-error';
+                break;
+            case 'Warning':
+                rowClass = 'tab-logs-data-warning';
+                break;
+            default:
+                rowClass = 'tab-logs-data-default';
+                break;
+        }
+        return rowClass;
+    }
+}
+
+export interface ILoggingProps {
+    request;
+    viewModel: ILoggingViewModel;
+}
+
+/**
+ * React class to for the console log messages tab
+ */
+export class Logging extends React.Component<ILoggingProps, {}> {
+    public getInitialState() {
+        return { checkedState: false };
```

Comment:
I think that's something I just copy/pasted from the request-detail-panel-execution tab:  https://github.com/Glimpse/Glimpse.Client.Prototype/blob/master/src/request/components/request-detail-panel-execution.jsx#l673.

If not used, it can be removed.